package.json

```diff
@@ -21,6 +21,7 @@
     "start-dev": "nodemon --inspect server.js",
     "prestart-dev": "npm run lint",
     "migrations": "sequelize db:migrate",
+    "unmigrations": "sequelize db:migrate:undo:all",
```

Comment:
This is a bit extreme :sweat_smile:. Maybe you can have a script to undo the last migration only. (it is not wrong either)